sdk/build.gradle

```diff
@@ -47,9 +47,10 @@ jar {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
     implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
-    implementation "com.squareup.okhttp3:okhttp:$okhttp_version"
-    implementation "com.fasterxml.jackson.module:jackson-module-kotlin:$jackson_version"
-    implementation "com.fasterxml.jackson.datatype:jackson-datatype-json-org:$jackson_version"
+    api "org.json:json:$json_version"
```

Comment:
Everything seems to compile without this line.

Reply:
travis build fail without this line